src/main/kotlin/blackjack/model/Cards.kt

```diff
@@ -0,0 +1,19 @@
+package blackjack.model
+
+class Cards(cards: List<Card>) {
+    private val cards = cards
+
+    fun score(): Int {
+        var sum = 0
+        cards.forEach {
+            sum = Denomination.sum(sum, it.denomination)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            sum = it.denomination.sum(sum)
```